### PR TITLE
ENH: Introduce manylinux option with ARM

### PR DESCRIPTION
**Pull request by tbirdso**
Exposes input parameter to allow the user to build for multiple Linux
toolsets and target architectures via available manylinux docker images.

Depends on ARM support introduced in ITKPythonPackage:
https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/236

### CODE REVIEW

**Review by thewtex — COMMENTED** [review]
Nice!

**Review by dzenanz — APPROVED** [review]

**Comment by tbirdso** [comment]
Updated with respect to #46 such that jobs are dynamically scheduled based on requested platforms. Linux jobs now produce one wheel per job.

Advantages of this approach:
- Fail-fast principle: initial wheel build can succeed more quickly. In ITKSplitComponents the first Linux job to run is (Python 3.7, _2_28, x64) and completes in approximately 5 minutes, whereas under the status quo running builds for _2_28-x64 + 2014-x64 + _2_28-aarch64 all in one job took approximately 30 minutes.
- It is obvious when one platform is failing, i.e. can tell at a glance if x64 builds pass but ARM fails
- Build outputs are cached more easily, i.e. if a URL request fails then it is less likely that earlier builds will have to be re-run.

Disadvantage of this approach:
- Easy to get the JSON input formatting wrong. Mitigated with README documentation and default parameter for reference. Also, the list of target platforms is not expected to be very long.
- Increases boilerplate, but not by very much. Linux job setup is typically quick and then each wheel build uses its own toolset and tarball, so very little was being reused between wheel builds for a given Python version under the status quo.

Testing at https://github.com/tbirdso/ITKSplitComponents/actions/runs/3840735170/jobs/6540141356 .

**Comment by tbirdso** [comment]
@dzenanz Good call. I have shortened to `build-linux-py` to preserve the "{verb}-{noun}" format for job titles.

**Comment by tbirdso** [comment]
@thewtex @dzenanz ping for re-review

**Review by dzenanz — APPROVED** [review]
I would prefer shortening Windows job names too, but this is good as is.

**Review by thewtex — APPROVED** [review]
Minor comment inline

File: .github/workflows/build-test-package-python.yml [hunk 1]
```diff
@@ -30,17 +30,24 @@ on:
         # example: InsightSoftwareConsortium/ITKMeshToPolyData@3ad8f08:InsightSoftwareConsortium/ITKBSplineGradient@0.3.0
         required: false
         type: string
+      manylinux-platforms:
+        description: 'JSON-formatted list of "<manylinux-image>-<arch>" specializations'
```
[the remark this answers is not in the document]
Comment (tbirdso) [reply .github/workflows/build-test-package-python.yml]: Updated here and in the README

**Pull request merged**